### PR TITLE
Public recipe list

### PR DESCRIPTION
**Pull request by jonas-45**
- [ ] In this branch, I added the 'generate shopping list' and 'add-ingredient' buttons to the recipe details page
- [ ] I also added the controller and views to add a new ingredient for a particular recipe
- [ ] I added the toggle button in the recipe details page with functionality that makes the recipe private or public on a toggle

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -38,27 +51,33 @@ def destroy
     @recipe.destroy
 
     respond_to do |format|
-      format.html { redirect_to recipes_url }
+      format.html { redirect_to recipes_url, notice: 'Recipe successfully removed' }
       format.json { head :no_content }
     end
   end
```
Comment (mertkantaroglu) [inline app/controllers/recipes_controller.rb:57]: Deleting the recipe function doesn't work at the moment. Besides, as an authenticated user, I shouldn't be able to see the remove button for other recipes and shouldn't be able to delete them. Please fix this part.

**Review by mertkantaroglu — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hello @jonas-45 👋 

You have done a very good job so far! 💯 👏  
On the other hand there are some minor errors you need to check. Please see them below.



## Requested Changes ♻️

File: app/controllers/recipes_controller.rb (hunk 1 above)
Comment (jonas-45) [reply app/controllers/recipes_controller.rb:57]: Got it. Resolving it right away

**Review by mertkantaroglu — APPROVED** [review]
## STATUS: APPROVED ✅ 

Hello @jonas-45 👋 

Good job on implementing the requested changes! 💯 

Your project is complete and you can merge it. Congrats! 👏  :shipit: 

minions

Congrats and Happy Coding! 🥂

**Pull request merged**